index.html

```diff
@@ -4,10 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    <h1>Hello Microverse!</h1>
+    <header>
+        <span ><a class="myLogo" href="#">Abedy</a></span>
+        <nav>
+            <a href="#">
+                <img src="icons/Union.png" alt="Union.png">
+            </a>
+        </nav>
+    </header>  
+    <section class="headline">
+        <h1 class="heading">I’m Abednego<br> Glad to see<br> you!</h1>
+        <main>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </main>
```

Comment:
- [ ] You did great work @AbednegoKibitok, however, you need to fix some important points 😃, Kindly make sure your website looks exactly like the design (`font`, `background image`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.). 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/234829547-3ac857b6-7877-4c13-9860-578fe44457b5.JPG)| |![Capture1](https://user-images.githubusercontent.com/57408419/234830228-032a57fc-a048-43db-9501-fbb462540a96.JPG)| |